README.md

```diff
@@ -1,2 +1,2 @@
 # EAL
-Employee Annual Leave Calculator
+Employee Annual Leave Calculator For Organizations
```

Comment:
It would Have been good if, the type of organizations that are fitted to use the application